content/integrate/redis-data-integration/ingest/reference/debezium/mysql.md

```diff
@@ -0,0 +1,15 @@
+---
+Title: RDI configuration for mysql
+linkTitle: mysql
+description: Describes the `application.properties` settings that configure Debezium Server for mysql
+weight: 
+alwaysopen: false
+categories: ["redis-di"]
+aliases:
+---
+
+## application.properties
+
+```properties
+
```

Comment:
Is this supposed to be empty? (Same with Oracle, PostgreSQL, and SQL Server config pages, btw.)